docs/hosting-a-server-with-northstar/troubleshooting.md

```diff
@@ -1,22 +1,5 @@
 # Troubleshooting
 
-## Document is empty
-
-`[error] Failed reading masterserver authentification response: encountered parse error 'The document is empty.'`
-
-Masterserver needs to request your gameserver for it to be authentified and registered.
-This error means that masterserver can't access your server's tcp port.
-
-Multiple problems can cause this error, but you can narrow it down by checking if your server is reachable from the outside.
-
-### Check if server is reachable
-
-You can check if the server is reachable using your internet browser.
-
-example : `http://{server_ip}:{server_tcp_port}/verify` should answer you `I am a northstar server!`
-
-Your server **must** be running while you check if the server is reachable.
```

Comment:
I wouldn't drop this section just yet but instead have a banner it only affects <`v1.13`

Keep in mind that currently `v1.13` only makes up 15% of servers.

Reply:
That section is misleading, outdated, and incorrect.

Reply:
You're right, the _"Document is empty"_ part is gone since switch to Atlas. The second section is still correct though, no? I mean it checks if server is up, not port forwarding and only applies to <`v1.13` but still...

Reply:
So keep or drop?

Reply:
The thing is, `/verify` is essentially useless IMO. It checks if you have the IP right, but it doesn't tell you anything about the UDP port or whether the game itself is actually in a working state... I'd say drop, but I can put back just the `/verify` stuff if you want.

Reply:
Aight, fine by me then ^^